src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.NoCom.cs

```diff
@@ -195,24 +195,11 @@ public static object GetUniqueObjectForIUnknown(IntPtr unknown)
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
-        public static bool IsComObject(object o)
-        {
-            if (o is null)
-            {
-                throw new ArgumentNullException(nameof(o));
-            }
+        public static bool IsComObject(object o) => false;
```

Comment:
Why these changes? I don't see a reason to remove the parameter checks in this case.

Reply:
yeah, I moved this to substitutions